scripts/skydb_v2.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Demo script for test all funktions from skydb v2.
+ *
+ * Example for Skydb v2 usage: node scripts/skydb.js dataKey "Seed Test Key" ./testdata/data.json
+ *
+ * Example with default data : node scripts/skydb.js dataKey  "Seed Test Key"
```

Comment:
```suggestion
 * Example with default data : node scripts/skydb_v2.js dataKey  "Seed Test Key"
```

Reply:
Sorry, I was too quick there. I just fixed it in the new version